tests/route_tester/base.py

```diff
@@ -16,6 +17,10 @@ def __init__(self, test, test_client):
     def endpoint(self):
         if not self._endpoint:
             self._endpoint = self.route.format(**self.params)
+            # This is specifically for the search route but I cannot see any draw
+            # backs with having this here.
+            if re.search("//", self._endpoint):
+                self._endpoint = re.sub("//", "/", self._endpoint)
```

Comment:
it does what I want.... os.path.normpath turns "/test/artifact/" to "/test/artifact" which will mess up our /<bucket-name>/artifact/<path:path> where path is left empty for a root search. I will look to see if there is a better built in solution but this does the job atm.